src/libraries/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -26,19 +26,19 @@ public class HttpWebRequest : WebRequest, ISerializable
 
         private WebHeaderCollection _webHeaderCollection = new WebHeaderCollection();
 
-        private readonly Uri _requestUri;
+        private readonly Uri _requestUri = null!;
```

Comment:
From public default constructor it should be nullable, but i guess that is not the intent

Reply:
The public default ctor was a mistake; code will get a compiler error if it tries to use it:
```C#
        [Obsolete("This API supports the .NET infrastructure and is not intended to be used directly from your code.", true)]
        [EditorBrowsable(EditorBrowsableState.Never)]
```
and despite the obsolete message, it's not even used by the implementation.

Reply:
Yeah i saw you added that for HttpWebResponse empty constructor, but not for this class, but seems that constructor not exposed in ref assembly so it must be fine, thanks